src/invertObject.js

```diff
@@ -18,8 +18,21 @@
  *
  * @return {Object}
  */
+
 function invertObject(items) {
-  // write code here
+  const invertedObj = {};
```

Comment:
what is `obj`? do you mean `object`?
make it easy
```suggestion
  const object = {};
```